shortapi/models.py

```diff
@@ -1,3 +1,28 @@
+from django.conf import settings
 from django.db import models
+from user.models import User
+from random import choices
+from string import ascii_letters
+
+
+
+class Link(models.Model):
+    user = models.ForeignKey(User,related_name="user",on_delete=models.CASCADE)
+    original_url = models.URLField()
+    shorten_url = models.URLField(blank=True, null=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    def shortener(self):
```

Comment:
separate this into a new file as a utility function or service that is without side effects i.e. no db calls